Home brewed to work with nss-db's on Solaris
community?

--- a/lib/util/util_paths.c	2020-08-05 14:12:29.543891939 +0000
+++ b/lib/util/util_paths.c	2020-08-05 14:13:06.588995937 +0000
@@ -26,6 +26,7 @@
 #include "dynconfig/dynconfig.h"
 #include "lib/util/util_paths.h"
 #include "system/passwd.h"
+#include <nss_dbdefs.h>
 
 /**
  * @brief Returns an absolute path to a file in the Samba modules directory.
